libfuzzer/libfuzz_harness.cc

```diff
@@ -10,27 +10,215 @@
 #include <string>
 #include <sstream>
 
+#include "asm_unmarshal.hpp"
+#include "crab_verifier.hpp"
+#include "platform.hpp"
+
 extern "C"
 {
+#define ebpf_inst ebpf_inst_ubpf
 #include "ebpf.h"
 #include "ubpf.h"
+#undef ebpf_inst
 }
 
 #include "test_helpers.h"
 #include <cassert>
 
+/**
+ * @brief Context structure passed to the BPF program. Modelled after the context structure used by XDP.
```

Comment:
```suggestion
 * @brief Context structure passed to the BPF program. Modeled after the context structure used by XDP.
```
Modelled is british.  Modeled is american.